apps/web/src/hooks/v3/useV3DerivedInfo.ts

```diff
@@ -176,6 +176,8 @@ export default function useV3DerivedInfo(
           : (invertPrice && typeof rightRangeTypedValue === 'boolean') ||
             (!invertPrice && typeof leftRangeTypedValue === 'boolean')
           ? tickSpaceLimits[Bound.LOWER]
+            ? tickSpaceLimits[Bound.LOWER] + 1
```

Comment:
for lower tick should be fine I think. Let's remove plus one here

Reply:
Done